### PR TITLE
add fileName and createdAt to Clinical resp + prettier

### PR DESCRIPTION
**Pull request by ciaranschutte**
**Description of changes**

add createdAt and fileName to clinical response 

**Type of Change**

- [ ] Bug
- [ ] Styling
- [x] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [ ] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: schemas/Clinical/index.js [hunk 1]
```diff
@@ -214,6 +214,8 @@ const convertRegistrationDataToGql = data => {
   return {
     id: data._id || null,
     creator: data.creator || null,
+    fileName: data.batchName || null,
+    createdAt: data.createdAt || null,
```
Comment (hlminh2000) [inline schemas/Clinical/index.js]: these are the only non-prettier changes right? if so, lgtm :)

**Comment by hlminh2000** [comment]
actually, do we need to update the program-service protobuff?

File: schemas/Clinical/index.js (hunk 1 above)
Comment (ciaranschutte) [reply schemas/Clinical/index.js]: yup!

**Comment by ciaranschutte** [comment]
@hlminh2000 needed to add extra fields to type

File: schemas/Clinical/index.js [hunk 2]
```diff
@@ -31,7 +31,8 @@ const typeDefs = gql`
   type ClinicalRegistrationData @cost(complexity: 10) {
     id: ID
     creator: String
-
+    fileName: String
+    createdAt: String
```
Comment (hlminh2000) [inline schemas/Clinical/index.js]: there's a `DateTime` scalar that should be used for date timestamps for consistency :)
See an example usage in: `schemas/Program/index.js`
Resolver for that scalar is in: `schemas/customScalars.js`

**Pull request merged**